.github/workflows/LocalTesting.yml

```diff
@@ -94,4 +94,50 @@ jobs:
           ICEBERG_SERVER_AVAILABLE: 1
           DUCKDB_ICEBERG_HAVE_GENERATED_DATA: 1
         run: |
+          make test_release
+
+      - name: Set up for Polaris
+        run: |
+          # install java
+          sudo apt install -y -qq openjdk-21-jre-headless
+          sudo apt install -y -qq openjdk-21-jdk-headless
+          # install python virtual environment (is this needed?)
+          sudo apt-get install -y -qq python3-venv
+          
+      - name: Setup Jenv
+        run: |
+          git clone https://github.com/jenv/jenv.git ~/.jenv
+          echo 'export PATH="$HOME/.jenv/bin:$PATH"' >> ~/.bash_profile
+          source ~/.bash_profile
+          eval "$(jenv init -)"
+          jenv enable-plugin export
+          exec $SHELL -l
+          jenv add /usr/lib/jvm/java-21-openjdk-amd64
+
+      - name: Setup Polaris
+        run: |
+          make setup_polaris
+          # let polaris initialize 
+          sleep 30
```

Comment:
I think this is definitely not long enough, we probably want some more reliable way of checking health of the polaris server

Reply:
```bash
          # Wait for Polaris to be healthy
          max_attempts=50
          attempt=1
          while ! (curl -sf http://localhost:8182/healthcheck || curl -sf http://localhost:8182/q/health); do
            if [ $attempt -gt $max_attempts ]; then
              echo "Polaris failed to initialize after $max_attempts attempts"
              exit 1
            fi
            echo "Waiting for Polaris to initialize (attempt $attempt/$max_attempts)..."
            sleep 5
            attempt=$((attempt + 1))
          done
          echo "Polaris is healthy"
```
This works locally for me